modules/stanford_earth_contact_footer/templates/block--stanford_earth_contact_footer.html.twig

```diff
@@ -0,0 +1,80 @@
+{#
```

Comment:
normally with template files we keep them as hyphens instead of underscores. Lets adjust those here and in the `hook_theme()` functions